scripts/minishift-setup.sh

```diff
@@ -0,0 +1,66 @@
+
+#!/bin/bash
+#
+#
+
+set -euo pipefail
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+cd ..
+
+# Lets try to detect the local machine ip..
+LOCAL_IP="$(dig $(hostname) +short)"
```

Comment:
yeah, this doesn't work for me here.  Maybe we should just leave this as an input, i.e. I don't mind setting it in my `.bashrc` for example.  On linux though there's `hostname -I | awk '{print $1;}'` but not sure that works on macs